shell/artifacts/Social/Particles.manifest

```diff
@@ -23,16 +23,27 @@ particle OnlyShowPosts in 'source/ShowPosts.js'
   description `show ${posts}`
 
 particle ShowSinglePost in 'source/ShowSinglePost.js'
-  ShowSinglePost(in Post post)
-  consume root
+  ShowSinglePost(in Post post, in [Avatar] avatars)
+  consume item
   description `show ${post}`
 
 particle WritePosts in 'source/WritePosts.js'
   WritePosts(inout Post post, inout [Post] posts)
   consume root
   description `write posts`
 
+// TODO(wkorman): Should we just use HostedParticleShape as
```

Comment:
If it works, yup. If it doesn't work, let me know and I'll try to figure out how to improve the type system so that it does.

Reply:
Sounds good, I'll look at this subsequently.